packages/react-component-library/src/components/Button/Button.mdx

```diff
@@ -0,0 +1,122 @@
+import { Meta } from '@storybook/blocks'
```

Comment:
I love this change in principle but wonder if there is a way to do this without losing the 'props table' and 'show code' features from dynamically generated docs page.

Maybe we could replace the existing Stories with these? Not only are these more succinct, they cover more permutations.